stoobly_agent/lib/api/body_param_names_resource.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import urllib
+import pdb
+
+from stoobly_agent.app.models.types import ParamNameCreateParams
+
+from ..logger import Logger
+from .endpoints_resource import EndpointsResource
+
+class BodyParamNamesResource(EndpointsResource):
```

Comment:
All the new methods in lib/api seem to be accessing fields of the passed in params assuming they'll be there such as `params['endpoint_id']}` which could return `None` so the call will fail.

The fix here would be to check if it exists at the start of the method and raise an exception if not